### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by ZhAlexR**

### CODE REVIEW

**Review by ykopatko — APPROVED** [review]

File: app/test_main.py [hunk 1]
```diff
@@ -1,3 +1,38 @@
...
 
-# write your code here
+age_values_data = [
+    (0, 0, [0, 0]),
+    (15, 15, [1, 1]),
+    (24, 24, [2, 2]),
+    (14, 16, [0, 1]),
+    (16, 14, [1, 0]),
+    (27, 27, [2, 2]),
+    (100, 59, [21, 9]),
```
Comment (Serhii-Leonenko) [inline app/test_main.py:11]: You don't test all cases, read the checklist

Comment (Serhii-Leonenko) [reply app/test_main.py:11]: Also you should have tests for all age boundaries

File: app/test_main.py [hunk 2]
```diff
@@ -1,3 +1,38 @@
+import pytest
 from app.main import get_human_age
```
Comment (Serhii-Leonenko) [inline app/test_main.py:2]: You should split different groups of imports with a blank line

**Review by Serhii-Leonenko — CHANGES_REQUESTED** [review]

**Review by Serhii-Leonenko — APPROVED** [review]
Well done!
